jans-cedarling/cedarling/src/lib.rs

```diff
@@ -24,36 +24,38 @@ mod models;
 #[cfg(test)]
 mod tests;
 
-use std::rc::Rc;
+use std::sync::Arc;
 
 use authz::Authz;
 use init::policy_store::{load_policy_store, LoadPolicyStoreError};
 pub use log::LogStorage;
 use log::{init_logger, LogWriter};
 pub use models::config::*;
-use models::log_entry::{LogEntry, LogType};
+pub use models::log_entry::LogEntry;
+use models::log_entry::LogType;
 use uuid7::uuid4;
 
 /// Errors that can occur during initialization Cedarling.
 #[derive(Debug, thiserror::Error)]
 pub enum InitCedarlingError {
     /// Error that may occur during loading the policy store.
-    #[error("Could not load policy :{0}")]
+    #[error("Could not load policy: {0}")]
     PolicyStore(#[from] LoadPolicyStoreError),
 }
 
 /// The instance of the Cedarling application.
+/// It is safe to share between threads.
 #[derive(Clone)]
 pub struct Cedarling {
     log: log::Logger,
     #[allow(dead_code)]
-    authz: Rc<Authz>,
+    authz: Arc<Authz>,
```

Comment:
Is `Authz` `Send` and `Sync`? If not, it'd be better to use `Rc`.
Another thing to consider is that will `Authz` be sent between threads, this is a pretty single-threaded application, so, there might not be a need for that.
Please let me know if I'm wrong.

Reply:
I made Cedarling to implement Send + Sync.
Authz it is not fully implemented for now. And we in the process of active development.

I need to implement Send + Sync to use Cedarling in the python bindings.

Reply:
Okay, no problemo.